carta/cpp/CartaLib/AxisDisplayInfo.h

```diff
@@ -0,0 +1,134 @@
+/***
+ * Information about how an axis should be displayed.
+ */
+
```

Comment:
It seems like that abstraction would be nice to have, provided @slovelan was building toward it.  But that's not where we are.  Since we have working code implemented in another direction, I'm concerned about abandoning this issue if it's going to represent a significant delay. Given the timelines involved in getting the rest of the work finished, can we live with this abstraction and move on?  Would building on the current state of affairs lead to harder maintenance and extensibility later during the line profiles and slices etc?